### PR TITLE
Query multiple tables

### PR DESCRIPTION
**Pull request by Chaseknis**
# Done so far

- Modify your inserted `animals` to include owner information 
- Modify your inserted `animals` so it includes the `species_id` value 
- Insert data into the `species` table
- Insert the following data into the `owners` table
- Modify `animals` table
- Create a table named `species`  
- Create a table named `owners` 
- Write queries using `JOIN`

## Screen-shots
Screenshot 2023-03-31 at 14 10 45

Screenshot 2023-03-31 at 14 10 25

Screenshot 2023-03-31 at 14 10 00

Screenshot 2023-03-31 at 14 09 41
Screenshot 2023-03-31 at 14 09 11
Screenshot 2023-03-31 at 14 08 47
Screenshot 2023-03-31 at 14 08 27

```
SELECT a.name
FROM animals AS a
JOIN owners AS o ON a.owner_id = o.id
WHERE o.full_name = 'Melody Pond';

SELECT a.name
FROM animals AS a
JOIN species AS s ON a.species_id = s.id
WHERE s.name = 'Pokemon';

SELECT o.full_name, a.name
FROM owners AS o
LEFT JOIN animals AS a ON o.id = a.owner_id;

SELECT s.name, COUNT(a.id) AS count
FROM animals AS a
JOIN species AS s ON a.species_id = s.id
GROUP BY s.name;

SELECT a.name
FROM animals AS a
JOIN owners AS o ON a.owner_id = o.id
JOIN species AS s ON a.species_id = s.id
WHERE o.full_name = 'Jennifer Orwell' AND s.name = 'Digimon';

SELECT a.name
FROM animals AS a
JOIN owners AS o ON a.owner_id = o.id
WHERE o.full_name = 'Dean Winchester' AND a.escape_attempts = 0;

SELECT o.full_name, COUNT(a.id) AS count
FROM owners AS o
JOIN animals AS a ON o.id = a.owner_id
GROUP BY o.full_name
ORDER BY count DESC
LIMIT 1;
```

### CODE REVIEW

**Review by Yerimah — CHANGES_REQUESTED** [review]
# STATUS: REQUESTED CHANGES 🔴 

Hi @Chaseknis, 👋  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

So Close Fall GIF by Archer


## Highlights
- Gitflow used ✔️ 
- Descriptive PR summary and title ✔️ 

## Required Changes ♻️

- [x] Great job 💪  with implementing some of the project requirements. However, you are expected to add screenshots of the query results and not screenshots of the queries like you have on your PR description. So, I will kindly suggest you add the query results from your SQL shell in place of the query images on your PR description.

database

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql [hunk 1]
```diff
@@ -92,3 +92,42 @@ WHERE date_of_birth >= '1990-01-01' AND date_of_birth <= '2000-12-31'
 GROUP BY species;
 
 SELECT COUNT(*) FROM animals WHERE escape_attempts = 0;
+
+
```
Comment (HAMMAS-SALEEM) [inline queries.sql:96]: image

**[OPTIONAL]**

- [x] In this query 11 records are returned but in actual 10 records should be returned 😨 You query is perfect 👍 but i guess there is something you missed from the previous milestone which is causing this issue 🤔 if you remember there was a transaction in the previous milestone which was deleting animals born after `2022-01-01` kindly run that transaction to keep your data up to date with the current milestone 👍

**Review by HAMMAS-SALEEM — APPROVED** [review]
# Status: Approved :green_circle: :1st_place_medal: :100:
success

### Hey @Chaseknis 👋

Great work on implementing the requested changes. Your project is complete. :blush:
There is nothing else to say about your project. :+1:

## Highlights :1st_place_medal:

✔️ Descriptive PR Title
✔️ Descriptive PR Summary
✔️ Following Gitflow
✔️ Following Requirements

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (Chaseknis) [reply queries.sql:96]: Thank you for the review, let me check and update the records

Comment (Chaseknis) [reply queries.sql:96]: Thank you @Yerimah

**Pull request merged**